docs/2.-Transaction-Examples.md

```diff
@@ -12,7 +12,7 @@ const { Api, JsonRpc } = require('eosjs');
 const { JsSignatureProvider } = require('eosjs/dist/eosjs-jssig');  // development only
 const fetch = require('node-fetch');                                // node only
 const { TextDecoder, TextEncoder } = require('util');               // node only
-const { TextEncoder, TextDecoder } = require('text-encoding');      // React Native, IE11, and Edge Browsers only
+const { TextEncoder, TextDecoder } = require('text-util');      // React Native, IE11, and Edge Browsers only
```

Comment:
Should this just be "require('util')"?

Reply:
ah dang no should've just been util. I'll update it in a later PR